util/constants.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Minimum deposit amount to activate an account
+ *
+ *  {@link https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-#:~:text=On%20the%20Polkadot%20network%2C%20an,the%20Existential%20Deposit%20(ED) What is the Existential Deposit}
+ */
+export const EXISTENTIAL_DEPOSIT_AMOUNT = 10_000_000_000n
```

Comment:
This is chain-specific, so ideally shouldn't be here. Can you move this constant to where it's used instead? Or put it in test util.